src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -53,41 +80,72 @@ export class GitHubAppsRegistry {
   // API for payload (JSON struct coming from GitHub)
 
   loadFromPayload(payload) {
-    // Soon we aim to support multiple orgs!
-    const org = '__tmp_org_placeholder__'; // payload?.organization?.login;
-    if (!org) {
-      throw new Error(`Could not find an org in ${JSON.stringify(payload)}.`);
+    // Org slug is differently accessed in org-scoped APIs vs. repo-scoped APIs
+    let org: string;
+    if (payload.organization) {
+      org = payload.organization.login;
```

Comment:
This has a net effect of doing away with some error handling we previously had, but I think it's okay. See https://github.com/getsentry/eng-pipes/pull/460/files#r1263058169.